buildlib/tools/test_mad.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+set -exE -o pipefail
+
+IMAGE="rdmz-harbor.rdmz.labs.mlnx/ucx/x86_64/rhel8.2/builder:mofed-5.0-1.0.0.0"
+cd "$BUILD_SOURCESDIRECTORY"
+
+node_setup() {
+    funcname
+    sudo chmod 777 /dev/infiniband/umad*
```

Comment:
This should be done by admin, not by test job

Reply:
And what if this folder gets recreated (reboot???)? test will fail, and we will need to manually chmod it.

Reply:
so maybe the test itself should run as root to be able to access this device node
in any case the test should not perform changes on the system

Reply:
As umad files are created dynamically, permissions must be set before running the test.
Otherwise we end up with errors:
[`UCX  ERROR mad_rpc_open_port(ca="mlx5_0" ca_port=1 mgmt_classes=IB_SA_CLASS) failed: Permission denied`](https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=79418&view=logs&j=fc80f01c-33c4-5912-cc2b-a3708338f199&t=0a6a587e-5e7b-5fe5-5647-b8ca39406e3c&l=19)

Reply:
the mad files should be created by openibd service, what do you mean by they are created dynamically?